File: push/src/main/java/com/xiaomi/push/log/MIPushDebugLog.java
package com.xiaomi.push.log;

import com.xiaomi.channel.commonutils.logger.LoggerInterface;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/push/log/MIPushDebugLog.class */
public class MIPushDebugLog implements LoggerInterface {
    private LoggerInterface sPushLogFileInterface;
    private LoggerInterface sUserLogInterface;

    public MIPushDebugLog(LoggerInterface loggerInterface, LoggerInterface loggerInterface2) {
        this.sUserLogInterface = null;
        this.sPushLogFileInterface = null;
        this.sUserLogInterface = loggerInterface;
        this.sPushLogFileInterface = loggerInterface2;
    }

    @Override // com.xiaomi.channel.commonutils.logger.LoggerInterface
    public void log(String str) {
        LoggerInterface loggerInterface = this.sUserLogInterface;
        if (loggerInterface != null) {
            loggerInterface.log(str);
        }
        LoggerInterface loggerInterface2 = this.sPushLogFileInterface;
        if (loggerInterface2 != null) {
            loggerInterface2.log(str);
        }
    }

    @Override // com.xiaomi.channel.commonutils.logger.LoggerInterface
    public void log(String str, Throwable th) {
        LoggerInterface loggerInterface = this.sUserLogInterface;
        if (loggerInterface != null) {
            loggerInterface.log(str, th);
        }
        LoggerInterface loggerInterface2 = this.sPushLogFileInterface;
        if (loggerInterface2 != null) {
            loggerInterface2.log(str, th);
        }
    }

    @Override // com.xiaomi.channel.commonutils.logger.LoggerInterface
    public void setTag(String str) {
    }
}

File: legacy/src/main/java/com/xiaomi/push/log/MIPushDebugLog.kt
package com.xiaomi.push.log
import io.github.magisk317.mipush.protocol.model.*

import com.xiaomi.channel.commonutils.logger.LoggerInterface

class MIPushDebugLog(
    private var sUserLogInterface: LoggerInterface?,
    private var sPushLogFileInterface: LoggerInterface?
) : LoggerInterface {
    override fun log(str: String) {
        sUserLogInterface?.log(str)
        sPushLogFileInterface?.log(str)
    }

    override fun log(str: String, th: Throwable) {
        sUserLogInterface?.log(str, th)
        sPushLogFileInterface?.log(str, th)
    }

    override fun setTag(str: String) {
        // No-op
    }
}
